### DISCUSSION TITLE
[Feature Request]: 支持不同国家的语言 /Supports languages ​​of different countries

### DISCUSSION THREAD

**Discussion by tuvinh0106** [jxxghp/MoviePilot#3345] [讨论]
### 当前程序版本

v1.3.1 or Lasted

### 功能改进类型

主程序

### 功能改进

你好
我希望你能支持英语和其他一些国家。
您可以根据 moviepilot 中的句子在 https://hosted.weblate.org/ 上打开该项目。我将依靠它来翻译成其他语言。

Hi,
I hope you will support English language and some other countries.
You can open the project on https://hosted.weblate.org/ Based on the sentences in moviepilot. I will rely on that to translate to other languages.

**Comment by xxww0098** [comment]
要走国际化了吗？？？

**Comment by tuvinh0106** [comment]
是的。我有家人在国外。但他们想观看英语节目以提高他们的语言水平。我希望能有所帮助。

**Comment by Shurelol** [comment]
没理解moviepilot有没有英文显示和整理播放的节目有没有英文之间的关系，国外的电影和电视节目大多不都是英文的么？
